### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by karimsan**
First of all, thank you for this package.

This PR includes updates to readme:
 - Aligned items in code blocks
 - Enabled syntax highlights in code blocks
 - Added concise description of `directAccess`
 - Added link to Google Cloud documentation section related to generating key file with credentials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,43 +65,49 @@ Alternatively, you can use
...
+  appId: 'my_app',
+  masterKey: 'master_key',
+  filesAdapter: gcsAdapter
+});
 ```
 
+## Options
+ - `directAccess` if set to `true`, uploaded files will be set as public and files will be served directly by Google Cloud Storage. Default is `false` and files are proxied by Parse server.
 
+ ## Obtaining credentials file
```
Comment (TomWFox) [inline README.md]:
```suggestion
## Obtaining credentials file
```

File: README.md [hunk 2]
```diff
@@ -63,43 +65,49 @@ Alternatively, you can use
...
+ Visit [Google Cloud documentation page](https://cloud.google.com/docs/authentication/production#obtaining_and_providing_service_account_credentials_manually) to see how to generate key file with credentials
```
Comment (TomWFox) [inline README.md]:
```suggestion
 Visit [Google Cloud documentation page](https://cloud.google.com/docs/authentication/production#obtaining_and_providing_service_account_credentials_manually) to see how to generate key file with credentials.
```

**Review by TomWFox — COMMENTED** [review]
Sorry this has been neglected for so long. I don't have experience with this adapter but these changes seem reasonable to me.

I couple minor fixes.

**Comment by karimsan** [comment]
@TomWFox thanks for the feedback, I've committed suggested changes.

**Review by davimacedo — APPROVED** [review]
It looks good to me

**Comment by TomWFox** [comment]
@karimsan now we have a good core team keeping tabs on things this sort of thing shouldn’t happen so I hope this experience doesn’t put you off making future contributions 🙂

And if ever we do miss things feel free to ping us on the forum or slack.

**Comment by karimsan** [comment]
@TomWFox thanks for merging. Everything is great, no worries at all, I am happy to deliver something valuable 👍

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [1.1.3](https://github.com/parse-community/parse-server-gcs-adapter/releases/tag/1.1.3)

**Pull request merged**